tests/datasource/fluent/integration/test_sql_datasources.py

```diff
@@ -653,77 +793,29 @@ def _is_quote_char_dialect_mismatch(
 @pytest.mark.parametrize(
     "column_name",
     [
+        # DDL: unquoted_lower_col ----------------------------------
```

Comment:
I'm having a lot of trouble reconciling the params with the overrides (e.g., expected to fail). It would be much more clear if we did a little bit of copy-paste here and params looked something like

`"datasource,column_name,expect_success", [param("snowflake", '"unquoted_lower_col"', False, id=...), ...]`

This lets me quickly figure out what should pass and what should fail

Reply:
As discussed in our last sync.
The raw queries test, and the `EXPECTED_FAILURE` mapping have been removed.

We determine whether or not the expectation should fail by doing a `SELECT <COLUMN_PARAM> from <TABLE> LIMIT 1;` before the checkpoint run.

If the raw query returns a result, then the column "exists" according to the specific SQL dialect and the subsequent validation result should succeed, if not it should fail.
This eliminates the maintenance burden and human error that could be introduced by maintaining these mappings and overrides.